docs/source/conf.py

```diff
@@ -35,7 +35,10 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ["sphinx.ext.intersphinx", "docs.source.html_builder"]
+extensions = ["sphinx.ext.intersphinx", "docs.source.html_builder", "myst_parser"]
+
+# To prevent myst-parser from warning header level increases
+suppress_warnings = ["myst.header"]
```

Comment:
Don't do this. Instead, decrease the indentation in `CHANGELOG.md`. Look at https://github.com/python/mypy/blob/72c413d/CHANGELOG.md?plain=1#L14 — it uses `####` (H4) right after `##` (H2). So MyST-Parser is correct to flag this. It's a real problem that needs to be fixed rather than ignored.

Reply:
Looks like it should be auto-replaced across that entire file. Running `sed -i 's/^####/###/g' CHANGELOG.md` should do what's needed to correct the mistake.

Reply:
Thank you very much for your guide!